vivisect/parsers/__init__.py

```diff
@@ -101,6 +101,14 @@ def getBytesParser(fmt):
     return None
 
 
+def align(v, alignment):
+    remainder = v % alignment
+    if remainder == 0:
+        return v
+    else:
+        return v + (alignment - remainder)
```

Comment:
duplication here with `envi.bits`. if its intended, ignore me.

Reply:
:)  read on, my friend.  you are named in my next commit.